src/app/page.tsx

```diff
@@ -1,13 +1,196 @@
+"use client";
+import { useEffect, useState } from "react";
+import { Textarea } from "@/components/ui/textarea";
 import { SampleServerComponent } from "@/sample/sampleAsync";
 import { Suspense } from "react";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+import { PlusCircle } from "@phosphor-icons/react";
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from "@/components/ui/command";
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+
+import {
+  Pagination,
+  PaginationContent,
+  PaginationEllipsis,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+} from "@/components/ui/pagination";
+
+import { Badge } from "@/components/ui/badge";
 
 export default function Home() {
   return (
     <main>
       {/* <Suspense fallback={<div>Loading...</div>}>
         <SampleServerComponent />
       </Suspense> */}
-      Hello World
+
+      <SearchBar />
+      <div className="m-4 md:m-10 mx-auto max-w-7xl grid gap-4 lg:grid-cols-3 md:grid-cols-2 grid-cols-1 ">
+        <NewCardComp />
+        <ProjectsCardComp
```

Comment:
Create a Projects List componenet, Add the new card and other cards in it, dont hardcode. use loop to render based on no. of total projects